app/Models/DatabaseDAO.php

```diff
@@ -21,7 +21,7 @@ public function create() {
 		try {
 			$sql = sprintf($SQL_CREATE_DB, empty($db['base']) ? '' : $db['base']);
 			return $this->pdo->exec($sql) !== false;
-		} catch (PDOException $e) {
+		} catch (Exception $e) {
```

Comment:
At the type of the writing, I visibly got confused by / forgot about the custom Minz_PDOConnectionException type, so the specific native catch was not working (I am personally not a fan of custom Exception types).

Reply:
Here, we would like to catch any type of error anyway